kvm/requests.yaml

```diff
@@ -3,3 +3,5 @@ releases:
       requests:
         - name: cert-exporter
           version: ">= 2.0.0"
+        - name: cert-operator
+          version: ">= 1.1.0 < 1.2.0"
```

Comment:
cert-operator 1.1.0 uses the config repo rather than draughtsman values.

I tested with 1.2.0 on Azure but cluster creation failed due to this check

> Add check to ensure that the Cluster resource is in the same namespace as the certConfig before creating the secret there.